src/main/java/nl/Tech/It/Easy/Tech/It/Easy/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,75 @@
+package nl.Tech.It.Easy.Tech.It.Easy.controllers;
+import nl.Tech.It.Easy.Tech.It.Easy.exceptions.RecordNotFoundException;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.ArrayList;
+import java.util.List;
+@RestController
+@RequestMapping("/tvs")
+public class TelevisionsController {
+
+    private List<Televisions> tvs = new ArrayList<>();
+
+    public TelevisionsController() {
+        // Add some TV data to the list
+        tvs.add(new Televisions(55.5, 32, "4K Ultra HD", true, true, false, true, 5));
+        tvs.add(new Televisions(45.0, 40, "1080p Full HD", true, true, false, false, 2));
+        tvs.add(new Televisions(60.0, 50, "8K Ultra HD", true, true, true, true, 3));
+    }
+
+    // Get all TVs
+    @GetMapping("/tv")
+    public ResponseEntity<List<Televisions>> getAllTvs() {
+        return ResponseEntity.ok(tvs);
+    }
+
+    // Get a TV by ID
+    @GetMapping("/tv/{id}")
+    public ResponseEntity<Televisions> getTvById(@PathVariable int id) {
+        Televisions tv = tvs.stream()
+                .filter(t -> t.getId() == id)
+                .findFirst()
```

Comment:
Stream is leuk om te gebruiken, maar het is vaak (zeker als je nog weinig programmeer ervaring hebt) makkelijker om het uit te schrijven in een for loop. 

In deze stream is de "findfirst" onnodig, odat de filter op id al een enkele unieke waarde zou moeten geven.